addons/calendar/static/src/views/attendee_calendar/common/attendee_calendar_common_popover.js

```diff
@@ -62,18 +62,18 @@ export class AttendeeCalendarCommonPopover extends CalendarCommonPopover {
      * @override
      */
     get isEventDeletable() {
-        return super.isEventDeletable && this.isCurrentUserAttendee && !this.isEventArchivable;
+        return super.isEventDeletable && this.isEventEditable && !this.isEventArchivable;
     }
 
     /**
      * @override
      */
     get isEventEditable() {
-        return this.isCurrentUserAttendee;
+        return this.props.record.rawRecord.user_can_edit;
```

Comment:
the code should work without user_can_edit in the view, the majority of installations will not update the calendar module and so the field will not be available for them

so I guess this should fallback on `this.isCurrentUserAttendee`

Reply:
Indeed, updated :+1: